docs/java-rest/high-level/supported-apis.asciidoc

```diff
@@ -5,6 +5,7 @@ The Java High Level REST Client supports the following APIs:
 
 Indices APIs::
 * <<java-rest-high-delete-index>>
+* <<java-rest-high-indices-exists>>
```

Comment:
I think that you may have forgotten to commit/push this new docs page.

Reply:
True.: )